### PR TITLE
Typos and man page fixes

### PR DESCRIPTION
**Pull request by guillemj**
Several typo and man page fixes.

----
## git request-pull output:
```
The following changes since commit bf3fedba890e66d644692910964fe1d8cbf4fb1b:

  test/xattr: don't rely on NUL-termination (2022-08-16 06:23:05 -0600)

are available in the Git repository at:

  https://github.com/guillemj/liburing pu/man-fixes

for you to fetch changes up to bf46399506adb26465f0da065b32d4318f9e7e68:

  man: Lowercase man page name in title header (2022-08-16 21:21:25 +0200)

----------------------------------------------------------------
Guillem Jover (7):
      Fix typo for "cancellation"
      Fix typo for "returns"
      Fix typo for "suppression"
      Fix typo for "successfully"
      Fix typo for "or"
      man: Add missing section numbers
      man: Lowercase man page name in title header

 CHANGELOG                       |  2 +-
 man/io_uring.7                  |  2 +-
 man/io_uring_cq_ready.3         |  4 ++--
 man/io_uring_enter.2            |  8 ++++----
 man/io_uring_free_probe.3       |  2 +-
 man/io_uring_get_probe.3        |  2 +-
 man/io_uring_opcode_supported.3 |  2 +-
 man/io_uring_prep_cancel.3      | 24 ++++++++++++------------
 man/io_uring_prep_poll_remove.3 |  6 +++---
 man/io_uring_prep_poll_update.3 |  6 +++---
 man/io_uring_register.2         |  2 +-
 man/io_uring_setup.2            |  2 +-
 man/io_uring_sq_ready.3         |  4 ++--
 man/io_uring_sq_space_left.3    |  4 ++--
 man/io_uring_sqe_set_flags.3    |  4 ++--
 man/io_uring_sqring_wait.3      |  2 +-
 src/include/liburing/io_uring.h |  4 ++--
 17 files changed, 40 insertions(+), 40 deletions(-)
```
----
<details>
<summary>Click to show/hide pull request guidelines</summary>

## Pull Request Guidelines
1. To make everyone easily filter pull request from the email
notification, use `[GIT PULL]` as a prefix in your PR title.
```
[GIT PULL] Your Pull Request Title
```
2. Follow the commit message format rules below.
3. Follow the Linux kernel coding style (see: https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst).

### Commit message format rules:
1. The first line is title (don't be more than 72 chars if possible).
2. Then an empty line.
3. Then a description (may be omitted for truly trivial changes).
4. Then an empty line again (if it has a description).
5. Then a `Signed-off-by` tag with your real name and email. For example:
```
Signed-off-by: Foo Bar <foo.bar@gmail.com>
```

The description should be word-wrapped at 72 chars. Some things should
not be word-wrapped. They may be some kind of quoted text - long
compiler error messages, oops reports, Link, etc. (things that have a
certain specific format).

Note that all of this goes in the commit message, not in the pull
request text. The pull request text should introduce what this pull
request does, and each commit message should explain the rationale for
why that particular change was made. The git tree is canonical source
of truth, not github.

Each patch should do one thing, and one thing only. If you find yourself
writing an explanation for why a patch is fixing multiple issues, that's
a good indication that the change should be split into separate patches.

If the commit is a fix for an issue, add a `Fixes` tag with the issue
URL.

Don't use GitHub anonymous email like this as the commit author:
```
123456789+username@users.noreply.github.com
```

Use a real email address!

### Commit message example:
```
src/queue: don't flush SQ ring for new wait interface

If we have IORING_FEAT_EXT_ARG, then timeouts are done through the
syscall instead of by posting an internal timeout. This was done
to be both more efficient, but also to enable multi-threaded use
the wait side. If we touch the SQ state by flushing it, that isn't
safe without synchronization.

Fixes: https://github.com/axboe/liburing/issues/402
Signed-off-by: Jens Axboe <axboe@kernel.dk>
```

</details>

----
## By submitting this pull request, I acknowledge that:
1. I have followed the above pull request guidelines.
2. I have the rights to submit this work under the same license.
3. I agree to a Developer Certificate of Origin (see https://developercertificate.org for more information).

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -13,7 +13,7 @@ liburing-2.2 release
 - Add support for registered ring fds, io_uring_register_ring_fd(),
   reducing the overhead of an io_uring_enter() system call.
 - Add support for the message ring opcode.
-- Add support for newer request cancelation features.
+- Add support for newer request cancellation features.
```
Comment (axboe) [inline CHANGELOG:16]: These aren't typos, both are valid spellings.

**Comment by axboe** [comment]
If you can drop the cancelation/cancellation patch, this looks good to me.

File: CHANGELOG (hunk 1 above)
Comment (guillemj) [reply CHANGELOG:16]: Ah, thanks, thought I had confirmed this after a "dict cancelation" returning nothing, but I see this is a common spelling in the US. Will drop, and file a report against lintian to not warn on these.

Comment (axboe) [reply CHANGELOG:16]: It is a US vs UK thing, both are valid spellings in the US. UK may only using the double ll variant, not sure.

**Comment by guillemj** [comment]
Ended up running codespell too, not sure whether you want an updated git request-pull output though?

**Comment by axboe** [comment]
> Ended up running codespell too, not sure whether you want an updated git request-pull output though?

Not necessarily, but perhaps we should just collapse these commits into one commit?

**Comment by guillemj** [comment]
Right, after the codespell fixes the number seemed excessive, so I've grouped them in a couple of commits, by tool that detected them. :)

**Review by ammarfaizi2 — APPROVED** [review]

**Pull request merged**